server/middleware/request.js

```diff
@@ -1,8 +1,21 @@
 const { v4: uuid } = require('uuid');
 const logger = require('../libs/logger');
-const { ValidationError } = require('../models/error');
+const { ValidationError, GenericError } = require('../models/error');
 const { isProduction } = require('../config');
 const listService = require('../services/list/');
+const { AxiosError } = require('axios');
+
+function axiosErrorMiddleware(err, _req, _res, next) {
+    if (err instanceof AxiosError) {
+        if (err.response) {
+            return next(new GenericError(err.response.data, err.response.status));
+        } else if (err.request) {
+            return next(new GenericError(`Failed to request following endpoint ${err.config.url}`, 500));
```

Comment:
Will this always be 500? Should we pass through `axiosError.status`?

Reply:
It looks like [AxiosError](https://github.com/axios/axios/blob/bdf493cf8b84eb3e3440e72d5725ba0f138e0451/lib/core/AxiosError.js) is fairly generic and will happen for errors below level 7. If an error occurs because of, say, `ECONNABORTED`, we need to pass that through to the logger.

We previously logged too much, but I suspect now we're logging far too little?

Reply:
Status is only available on `AxiosResponse` object which you don't have if its not got that far (https://axios-http.com/docs/handling_errors). 

I do think there is a good call here to add the Axios code which contains `ECONNABORTED`.

Reply:
This has been rectified.